Source/Shaders/Appearances/PolylineColorAppearanceVS.glsl

```diff
@@ -1,3 +1,12 @@
+#ifdef VECTOR_TILE
```

Comment:
Is this uber shader better than having two separate shaders and perhaps shared GLSL functions as needed?